lib/stormpath-sdk/resource/collection.rb

```diff
@@ -51,6 +51,10 @@ def limit limit
     self
   end
 
+  def size
+    current_page.items.size
```

Comment:
As i understood from the documentation http://docs.stormpath.com/rest/product-guide/#collection-resources size is a method on collection (An integer that represents the total number of items in the collection)

> This should be a property of the CollectionPage class, just like offset, limit and items are

aren't offset, limit and order methods on Collection class, not properties on CollectionPage?

This is a commit regarding the following issue https://github.com/stormpath/stormpath-sdk-ruby/issues/68

Can you please clarify what needs to be changed here and how the size method should behave?